dhall/src/Dhall/Import.hs

```diff
@@ -986,6 +990,12 @@ loadRelativeTo :: FilePath -> Expr Src Import -> IO (Expr Src X)
 loadRelativeTo rootDirectory expression =
     State.evalStateT (loadWith expression) (emptyStatus rootDirectory)
 
+loadWithoutCacheRelativeTo :: FilePath -> Expr Src Import -> IO (Expr Src X)
```

Comment:
Perhaps change `loadRelativeTo` to take a `SemanticCacheMode` argument?  After all, it hasn't been part of the public API yet so it wouldn't be a breaking change to do so